poetry.lock

```diff
@@ -213,22 +230,19 @@ files = [
 
 [[package]]
 name = "geoalchemy2"
-version = "0.14.2"
+version = "0.13.3"
```

Comment:
Was there a particular reason to downgrade geoalchemy?

Reply:
I didn't do that purposely or directly. Evidently Poetry's resolution phase decided that was the compatible version given some change I made. I'll see if I can track it down.

Reply:
Pycds 4.4.0 specifies a particular version, namely `geoalchemy2 = "^0.13.3"`. This was part of [work](https://github.com/pacificclimate/pycds/pull/164) to clean up ([rationale](https://github.com/pacificclimate/pycds/issues/162)) the conversion from Pipenv to Poetry. [geoalchemy2](url) latest [was 0.13.3](https://pypi.org/project/GeoAlchemy2/#history) at the time of pycds 4.4.0.

A contribution to this possibly undesirable situation may be that for versions numbered `0.x.y`, [the minor version is treated like a major version, etc.](https://python-poetry.org/docs/dependency-specification/#caret-requirements). This is I believe a SemVer standard, not a Poetry idiosyncracy.

We can upgrade geoalchemy2 in future versions of pycds; 

Further thoughts?